typings/Command.ts

```diff
@@ -0,0 +1,31 @@
+interface Choice {
```

Comment:
typing folder is for lib types only

Reply:
Ok I will change this later, in this pull request it doesn't make sense